server/controllers/documentController.js

```diff
@@ -1,132 +1,298 @@
 import db from '../models';
 
 const Documents = db.documents;
+const Users = db.users;
 
+/**
+ * Controller for document management
+ */
 class DocumentController {
-    /**
-     * Method to set the various document routes
-     * @param{Object} app - Express app
-     * @return{Void}
-     */
-    static postRequest(request) {
-        return (
-            request.body &&
-            request.body.title &&
-            request.body.content &&
-            request.body.access &&
-            request.body.OwnerId
-        )
+  /**
+   * Method to set the various document routes
+   * @param{Object} request - Server request
+   * @return{Void} return Void
+   */
+  static postRequest(request) {
+    return (
+      request.body &&
+      request.body.title &&
+      request.body.content &&
+      request.body.access &&
+      request.decoded.UserId
+    );
+  }
+  /**
+   * Method used to create new Document for a particular user
+   * @param{Object} request - Server Request
+   * @param{Object} response - Server Response
+   * @returns{Void} return Void
+   */
+  static createDocument(request, response) {
+    if (DocumentController.postRequest(request)) {
+      Documents
+        .create({
+          title: request.body.title,
+          content: request.body.content,
+          access: request.body.access ? request.body.access : 'public',
+          OwnerId: request.decoded.UserId,
+        })
+        .then((document) => {
+          response.status(201).send({
+            success: true,
+            message: 'Document successfully created',
+            document: document.dataValues
+          });
+        })
+        .catch((error) => {
+          response.status(401).send({
```

Comment:
401 error status code means the user is unauthorized, do you really think that is what happens at this point?